### PR TITLE
Fixed some bugs

### PR DESCRIPTION
**Pull request by juliangarritano**
Now blurring button after mouseup.
Now wrapping buttons that span more than one line.

### CODE REVIEW

**Comment by lemieux** [comment]
:+1:

**Comment by mspensieri** [comment]
:+1:

**Comment by dannytranlx** [comment]
:+1:

File: src/stylesheets/conversation.less [hunk 1]
```diff
@@ -190,5 +182,4 @@
             }
         }
     }
-}
-
+}
```
Comment (dannytranlx) [inline src/stylesheets/conversation.less]: @lemieux Doesn't the new csscomb add those usually? Are we all using the same formatter?

Comment (lemieux) [inline src/stylesheets/conversation.less]: ask @juliangarritano if he is using it first :p

Comment (juliangarritano) [inline src/stylesheets/conversation.less]: I am not using csscomb/
Dafuq is that?

Comment (juliangarritano) [inline src/stylesheets/conversation.less]: Should I be using csscomb?

Comment (juliangarritano) [inline src/stylesheets/conversation.less]: I don't really like the spacing it does much

**Pull request merged**